lms/djangoapps/teams/static/teams/js/views/topic_teams.js

```diff
@@ -32,9 +33,10 @@
                 render: function() {
                     var self = this;
                     this.collection.refresh().done(function() {
+                        var messageHtml;
                         TeamsView.prototype.render.call(self);
                         if (self.canUserCreateTeam()) {
-                            var message = interpolate_text(
+                            messageHtml = HtmlUtils.interpolateHtml(
```

Comment:
It seems like very complex logic to have live in a template. I prefer to minimize the logic in templates because they don't count towards code coverage so it is hard to know how well it is being tested. It is also much harder to debug code inside a template.